.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,14 +1,17 @@
 blank_issues_enabled: false
 contact_links:
   - name: Discussions
-    url: https://github.com/scikit-learn/scikit-learn/discussions
+    url: https://github.com/scikit-learn/scikit-learn/discussions/new
     about: Ask questions and discuss with other scikit-learn community members
-  - name: Stack overflow
+  - name: Stack Overflow
     url: https://stackoverflow.com/questions/tagged/scikit-learn
-    about: Please ask and answer usage questions on stackoverflow
+    about: Please ask and answer usage questions on Stack Overflow
   - name: Mailing list 
     url: https://mail.python.org/mailman/listinfo/scikit-learn
     about: General discussions and announcements on the mailing list
   - name: Gitter
     url: https://gitter.im/scikit-learn/scikit-learn
     about: Users and developers can sometimes be found on the gitter channel
+  - name: Blank issue
+    url: https://github.com/scikit-learn/scikit-learn/issue/new
+    about: Github Discussions should be used in most cases, unless you know better
```

Comment:
The last part feels a little uninviting:

```suggestion
    about: Github Discussions should be used in most cases
```

Reply:
I ended up using: Please note that Github Discussions should be used in most cases instead. Hopefully that's good enough